tests/unit_tests/example_test/Makefile.am

```diff
@@ -0,0 +1,13 @@
+AUTOMAKE_OPTIONS = foreign
+
+check_PROGRAMS = testdriver testdriver2
```

Comment:
Maybe rename them to _example_testdriver_ and _example_testdriver2_ because they are really just that. We can even delete them when we have enough "real" tests that make a template superfulous.